conversion/conversion.go

```diff
@@ -83,7 +83,7 @@ const (
 
 var (
 	// Set the maximum number of concurrent workers during foreign key creation.
-	MaxWorkers = 10
+	MaxWorkers = 20
```

Comment:
I remember you mentioned some caveats around changing this to a very high number e.g., 100. It might be worth adding a comment describing how to set this value and how it relates to database admin QPS quota.